releasenotes.txt

```diff
@@ -1,7 +1,2 @@
-release notes for cppcheck-2.10
+release notes for cppcheck-2.11
 
-- the deprecated Makefile option SRCDIR is no longer accepted
-- if the file provided via "--file-list" cannot be opened it will now error out
-- add command-line option "--disable=<id>" to individually disable checks
-- added CMake option BUILD_CORE_DLL to build lib as cppcheck-core.dll with Visual Studio
-- Windows binaries currently default to the "win32A" and "win64" platform respectively. Starting with Cppcheck 2.13 they will default to 'native' instead. Please specify '--platform=win32A' or '--platform=win64' explicitly if you rely on this.
```

Comment:
Why not stack the release notes for each release rather removing the old notes in favor of the new one?

Reply:
The releasenotes are not well maintained. We just recently got into documenting things a bit better. But is is probably still subpar. @danmar usually provides some brief write-up on the GitHub/Trac release page but that varies greatly.

As we are doing a lot of work based on tickets and mainly do changes via PRs you have some implicit documentation based on that in GitHub and in Trac - see https://trac.cppcheck.net/milestone/2.10 for the current stable release.